viv_utils/flirt.py

```diff
@@ -14,6 +20,14 @@
 _LIBRARY_META_KEY = "is-library"
 
 
+@contextlib.contextmanager
```

Comment:
this and the functions below are almost verbatim taken out of capa (removed there)